pyflow/RCDT2/Nodes/node_loader.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: Alliander N. V.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from RCDT2.Nodes.core import Service
+
+
+def get_pyflow_nodes_from_ros_services(services: list[tuple[str, str]]) -> dict:
```

Comment:
Deze type hint is niet heel lekker te lezen. In de toekomst zou je kunnen kijken om een custom type te definieren zodat je iets als `list[serviceDefinition]` krijgt